README.md

```diff
@@ -117,8 +119,32 @@ pod install
 
 The SDK must be initialized with the short alphanumeric
 [public API key](https://help.klaviyo.com/hc/en-us/articles/115005062267#difference-between-public-and-private-api-keys1)
-for your Klaviyo account, also known as your Site ID. Initialization is done in the native layer, and must occur before
-any other SDK methods can be invoked. Follow the native SDK instructions for initialization, and refer to the
+for your Klaviyo account, also known as your Site ID.
+
+Initialization can either be done from your app's React native or native code, and must occur before
+any other SDK methods can be invoked so that Klaviyo SDK can track profiles, events and push tokens
+towards your company.
+
+The decision to initialize the SDK from the React Native or the native code depends on your app's architecture.
+
+Finally, the SDK initialization needs to be done just once either in your react native or native code.
```

Comment:
I'd take out "finally"

Reply:
In fact I think I'd take out this whole sentence. If anything, you could say more in the sentence above, but I think this is quite clear:

> Initialization can either be done from your app's React native or native code